jamjar/jamjar/concerts/serializers.py

```diff
@@ -107,3 +111,10 @@ def get_graph(self, obj):
 
     def get_videos_count(self, obj):
         return obj.videos.count()
+
+    def get_jamjars_count(self, obj):
+        data = JamJarMap.objects.filter(video__concert__id=obj.id).values('video__concert_id').annotate(num_jamjars=Count('start_id', distinct=True))
```

Comment:
Will this be accurate since it looks like it'll include JamJars that are just a single video (video == jamstart)?